code/style.css

```diff
@@ -5,3 +5,368 @@ this can be used for the FAQ-accordion. read more here: https://www.w3schools.co
 .active + .answer {
   /* some styling here */
 }
+
+body {
+margin: 0;
+color: #888888;
+
+}
+
+
+/*style the header*/
+.header{
+  width: 100%;
+  height: 40vh;
+  overflow: hidden;
+
+}
+
+
+.header h1 {
+  color: #D8E8FF;
+  text-shadow: 2px 2px 5px #888888;
+ font-size: 70px;
+
+
+}
```

Comment:
Remove empty lines inside the brackets but really nice to see the structure with body on top and then header 💫